sb.web.app/src/components/Profile/MyPurchases.vue

```diff
@@ -122,17 +122,23 @@
       </table>
       </div>
     </div>
-    
+    <LoadingComponent ref="loadingComponent" />
+
 </template>
 
 <script>
-import { getPurchases, cancelRequest, acceptRequest, postNotification, getAvailableBalance, getUserAvailableBalance, getLimits } from '../../serverFetch'
+import { getPurchases, cancelRequest, acceptRequest, postNotification, getAvailableBalance, getUserAvailableBalance, getLimits, setTransactionsData } from '../../serverFetch'
```

Comment:
not important but we should remove getPurchases from here if not used any more - maybe even from serverFetch?